Jenkinsfile.demo

```diff
@@ -13,9 +13,9 @@ def gitEmail() {
 }
 
 def ipAddress() {
-    sh "docker inspect test-container-${env.BUILD_NUMBER} | jq -r '.[0].NetworkSettings.IPAddress' > IP_ADDRESS-${env.BUILD_NUMBER}"
-    def ipAddress = readFile("IP_ADDRESS-${env.BUILD_NUMBER}").trim()
-    sh "rm -f IP_ADDRESS-${env.BUILD_NUMBER}"
+    sh "docker inspect test-container | jq -r '.[0].NetworkSettings.IPAddress' > IP_ADDRESS"
```

Comment:
Why remove the build number?

Reply:
It was causing the rebuild to fail

Reply:
By removing the build number I can clean up the job easily at the end of the pipeline